crates/rpc-types-engine/src/version.rs

```diff
@@ -0,0 +1,16 @@
+//! Versions for the engine api.
+
+/// The version of the engine api.
+#[derive(Debug, Clone, Copy, PartialEq, Eq, PartialOrd, Ord, Hash)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+#[repr(u32)]
+pub enum ForkchoiceUpdateVersion {
+    /// Version 1 of the engine api.
+    V1 = 1,
+    /// Version 2 of the engine api.
+    V2 = 2,
+    /// Version 3 of the engine api.
+    V3 = 3,
+    /// Version 4 of the engine api.
+    V4 = 4,
```

Comment:
there's actually no fcu 4 for prague anymore, only newPayloadv4, so we either remove this or rename it to something like

https://github.com/paradigmxyz/reth/blob/9d5e159968737950473a572cdce3881af033f9fc/crates/payload/primitives/src/lib.rs#L324-L342